src/router.js

```diff
@@ -40,6 +40,9 @@ export default new Router({
         header: AppHeader,
         default: Login,
         footer: AppFooter
+      },
+      props: {
+        default: true
```

Comment:
what is the default for

Reply:
default maps to the default component (line 41) and means it takes router params as its props